### PR TITLE
docs(backend): added fix for intel macbooks to run tigerbeetle

### LINKED ISSUE #341: Tests don't run on 2018 macbook pro intel
Currently running the backend tests is broken on my computer. I have a 2018 macbook pro.

When I run ```yarn workspace backend test``` I get the following output 
image

 I traced the issue to the tigerbeetle docker images attempting to be run in on this [line](https://github.com/interledger/rafiki/blob/557fe12851bb7d34911dcba13d8724cee0403f83/packages/backend/jest.setup.js#L71). 

If you attempt to start the image by itself you get 
image

There is an [issue](https://github.com/coilhq/tigerbeetle/issues/92) in the tigerbeetle repo referencing this same problem.

### PR DESCRIPTION
**Pull request by tyleralt**
fixes #341

## Changes proposed in this pull request
- Changed backend docs to cover known issue with testing.

## Context

this [issue](https://github.com/interledger/rafiki/issues/341) covers the testing issue present on some intel based macbooks while trying to run tigerbeetle. 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

**Comment by tyleralt** [comment]
@wilsonianb Does this look good?

File: packages/backend/README.md [hunk 1]
```diff
@@ -5,6 +5,20 @@
 ### Prerequisites
 
 - [Docker](https://docs.docker.com/engine/install/) configured to [run as non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+- If you are running MacOS on a non apple silicon device there is a known [issue](https://github.com/coilhq/tigerbeetle/issues/92) with
+  [tigerbeetle](https://github.com/coilhq/tigerbeetle). To fix this you can set defaults in `$HOME/.docker/daemon.json` with
+
+  ```
```
Comment (sublimator) [inline packages/backend/README.md:11]: It's not really an issue with TB as such. It requires what it requires. 
It's more a lacking in the [testcontainers](https://github.com/testcontainers/testcontainers-node)  package

Comment (sublimator) [reply packages/backend/README.md:11]: There's no API to add linux capabilities or set the ulimits

Comment (tyleralt) [reply packages/backend/README.md:10]: That makes sense. I updated the wording to better reflect the problem. Thanks!

Comment (tyleralt) [reply packages/backend/README.md:10]: @sublimator Does it seem reasonable with the new edits?

Comment (sublimator) [reply packages/backend/README.md:10]: I'm not really sure @tyleralt. Note that I'm not a rafiki dev

I guess you want to say something like this:
```
TigerBeetle requires the privilege of using memlock functions, usually afforded
by adding the linux capability (IPC_LOCK). Unfortunately, rafiki uses
[testcontainers](https://github.com/testcontainers/testcontainers-node) which
provides no api to configure the containers as so. On MacOS, a workaround is
to ... 
```

To be honest I'm not sure why it seems to work on linux (core Rafiki devs and
Github actions CI). Perhaps linux docker has different defaults ? 

@wilsonianb  

What do you think?

**Comment by matdehaast** [comment]
Thanks for this @tyleralt this is good.

**Review by matdehaast — APPROVED** [review]

File: packages/backend/README.md [hunk 2]
```diff
@@ -5,6 +5,19 @@
...
+  "default-ulimits": {
+    "memlock": {
+      "Hard": -1,
+      "Name": "memlock",
+      "Soft": -1
+    }
+  },
+  ```
+
+  and then restart docker.
```
Comment (wilsonianb) [inline packages/backend/README.md:20]: It looks like prettier wants an extra line
```suggestion
  and then restart docker.

```

**Review by wilsonianb — APPROVED** [review]

**Pull request merged**